tests/generics.rs

```diff
@@ -3,7 +3,10 @@ use snafu::{prelude::*, Backtrace};
 type BoxError = Box<dyn std::error::Error>;
 
 #[derive(Debug, Snafu)]
-enum Error<'a, 'x, A, Y> {
+enum Error<'a, 'x, A, Y>
+where
```

Comment:
No where clauses here; instead add `Whatever` variants down in the `mod inline` / `mod where_clause` sections below.

Reply:
Ah, sorry about that. Done.